src/mlir/dialect/VeronaOps.td

```diff
@@ -244,4 +245,117 @@ def Verona_FieldOp : Verona_Op<"field", [InClassScope]> {
     let verifier = [{ return success(); }];
 }
 
+def Verona_ContinueOp : Verona_Op<"continue", [Terminator, InLoopScope]> {
+    let summary = "continues any loop block";
+
+    let arguments = (ins);
+    let results = (outs);
+    let assemblyFormat = [{ attr-dict }];
+    let verifier = [{ return success(); }];
+}
+
+def Verona_BreakOp : Verona_Op<"break", [Terminator, InLoopScope]> {
+    let summary = "breaks any loop region, returning control to previous region";
+
+    let arguments = (ins);
+    let results = (outs);
+    let assemblyFormat = [{ attr-dict }];
+    let verifier = [{ return success(); }];
+}
+
+def Verona_LoopExitOp : Verona_Op<"loop_exit", [InLoopScope]> {
+    let summary = "breaks any loop block if condition is `false`";
+
+    let builders = [
+      OpBuilder<"OpBuilder &builder, OperationState &state, bool cond">
```

Comment:
Is this builder actually used / defined anywhere?

Reply:
Hmm, good question. I've added and removed a lot of code, this could be a left over. Will check.